boa/src/builtins/string/mod.rs

```diff
@@ -1329,6 +1335,42 @@ impl String {
         RegExp::match_all(&re, this.to_string(context)?.to_string(), context)
     }
 
+    /// `String.prototype.normalize( [ form ] )`
+    ///
+    /// The normalize() method normalizes a string into a form specified in the Unicode® Standard Annex #15
+    ///
+    /// More information:
+    ///  - [ECMAScript reference][spec]
+    ///  - [MDN documentation][mdn]
+    ///
+    /// [spec]: https://tc39.es/ecma262/#sec-string.prototype.normalize
+    /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/normalize
+
+    pub(crate) fn normalize(this: &Value, args: &[Value], context: &mut Context) -> Result<Value> {
+        let this = this.require_object_coercible(context)?;
+        let s = this.to_string(context)?;
+        let form = args.get(0).cloned().unwrap_or_default();
+
+        let f_str;
+
+        let f = match form.is_undefined() {
+            true => "NFC",
+            false => {
+                f_str = form.to_string(context)?;
+                f_str.as_str()
+            }
+        };
```

Comment:
It's a bit strange to match a boolean, right? I would just use if/else.

Reply:
Yeah I had it as if/else before, but changed it to see how it'd change lifetimes if at all, but I agree, probably more clear with an if/else.